src/azure-cli/azure/cli/command_modules/appservice/tests/latest/recordings/test_functionapp_slot_appsetting_update.yaml

```diff
@@ -1,32 +1,27 @@
 interactions:
 - request:
-    body: '{"location": "westus", "tags": {"product": "azurecli", "cause": "automation",
-      "date": "2019-06-19T22:02:05Z"}}'
+    body: null
     headers:
       Accept:
       - application/json
       Accept-Encoding:
       - gzip, deflate
       CommandName:
-      - group create
+      - functionapp plan create
```

Comment:
As we can see here and [here](https://github.com/Azure/azure-cli/pull/9747/files#diff-8ed9aaa34535ded5e7137146ba395147R56) the first few requests is not `group create` or `storage account create` but `functionapp plan create`